charm/src/iframe/prompt.ts

```diff
@@ -31,21 +31,20 @@ export const buildPrompt = ({
   schema: JSONSchema;
   model?: string;
 }): LLMRequest => {
-  const messages = [];
+  const messages: string[] = [];
   if (spec && src) {
     messages.push(spec);
-    messages.push("```html\n" + src + "\n```");
+    messages.push("```javascript\n" + extractUserCode(src) + "\n```");
   }
 
   messages.push(
-    `The user asked you to ${
-      spec ? "update" : "create"
+    `The user asked you to ${spec ? "update" : "create"
```

Comment:
this phrase (that I wrote) maybe could be improved...  it seems silly to read now